# 以相機座標系統到無人機座標系統的轉換，單位是公尺及度
# 以無人機為原點, 但以像素座標系統為基準, x軸向右, y軸向後, z軸向上
camera_offset:
  x: 0.0
  y: 0.125
  z: 0.0
  roll: 0.0
  pitch: 0.0
  yaw: 180.0
camera_parameter:
  # 相機內部參數
  camera_matrix:
    - [184.752, 0.0, 320.0]
    - [0.0, 184.752, 240.0]
    - [0.0, 0.0, 1.0]
  distortion_coefficients: [0.0, 0.0, 0.0, 0.0, 0.0]
